### DISCUSSION TITLE
[@types/react] (^18) useDeferredValue JSDoc outdated

### DISCUSSION THREAD

**Discussion by GalvinGao** [DefinitelyTyped/DefinitelyTyped#60564] [Issues with a @types package]
> Originally posted on https://github.com/facebook/react/issues/24627
---
React version: `^18.0.0` (`18.0.0`)

## The current behavior
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/19b70b2c6cbd626b1f861d4eef0954b0bf3fa5a5/types/react/index.d.ts#L1119-L1121 currently still references
> Returns a deferred version of the value that may “lag behind” it for at most `timeoutMs`.

which is outdated due to the fact `timeoutMs` is [already removed](https://github.com/facebook/react/pull/19703).

## The expected behavior
It shows the correct documentation JSDoc as mentioned in [the official React documentation](https://17.reactjs.org/docs/concurrent-mode-reference.html#usedeferredvalue)
> Returns a deferred version of the value that may “lag behind” it.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
I just opened #60567
